server/src/infra/sql/smart.info.repository.sql

```diff
@@ -107,7 +107,7 @@ WITH
     WHERE
       "asset"."ownerId" IN ($2)
     ORDER BY
-      1 + ("faces"."embedding" <= > $1) ASC
+      1 + ("faces"."embedding" <= > $3) ASC
```

Comment:
I think this is wrong, it should be `$1`, so it's probably a regression.

Reply:
Fixed with reverted package lock